kubetest/e2e.go

```diff
@@ -541,13 +541,15 @@ func dumpFederationLogs(location string) error {
 	return nil
 }
 
-func perfTest() error {
+func perfTest(args string) func() error {
 	// Run perf tests
-	cmdline := util.K8s("perf-tests", "clusterloader", "run-e2e.sh")
-	if err := control.FinishRunning(exec.Command(cmdline)); err != nil {
-		return err
+	return func() error {
+		cmdline := util.K8s("perf-tests", "run-e2e.sh")
+		if err := control.FinishRunning(exec.Command(cmdline, args)); err != nil {
```

Comment:
Can you please confirm if this will work - that you're passing all args as a single string instead of an array. Because I remember the latter option being used at various places in the code.

Reply:
It turned out there was a problem with this implementation. I've changed it, should now work fine.